hack/gotest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+CGO_ENABLED=0
```

Comment:
You may need to `export` this or move it to the `go test ...` line:

```sh
$ ABC=1
$ env | grep ABC
$ ABC=1 env | grep ABC
ABC=1
$ export ABC=1
$ env | grep ABC
ABC=1
```

Reply:
whoops, yes thanks